app/controllers/v1/forams_controller.rb

```diff
@@ -1,7 +1,7 @@
 module V1
   class ForamsController < ApplicationController
     before_action :set_foram, only: :show
-
+    before_action :authenticate_user!
```

Comment:
Wynieśmy do klasy bazowej `V1::BaseController` i wprowadźmy dziedziczenie we wszystkich  pozostałych zasobach.